ui/homework/homework_1/test1.ts

```diff
@@ -4,7 +4,10 @@
 */
 
 type Person = {
+  name: string;
+  age: number | 'unknown';
```

Comment:
Are you sure about this type?

Reply:
I'm not really sure. Should there be "age: number | string" here?